src/app/containers/App/App.jsx

```diff
@@ -88,13 +99,17 @@ export const App = ({ location, initialData, bbcOrigin, history }) => {
     }
   }, [routeHasChanged]);
 
-  return renderRoutes(routes, {
-    ...state,
-    bbcOrigin,
-    previousPath,
-    loading: routeHasChanged,
-    showAdsBasedOnLocation,
-  });
+  return (
+    <StyledApp darkMode={initialData.darkMode}>
```

Comment:
Not one for now but I think this sort of thing could be controlled with a theme context provider

Reply:
I've added that to the refactor notes. 👍  P.S. This logic was moved to `src/app/Layouts/defaultPageWrapper.jsx` at the end of yesterday following the merge of Ryan's PR.